examples/tv-casting-app/linux/simple-app-helper.cpp

```diff
@@ -28,11 +29,14 @@
 #include <lib/support/CHIPMem.h>
 #include <lib/support/CodeUtils.h>
 #include <platform/CHIPDeviceLayer.h>
+#include <platform/TestOnlyCommissionableDataProvider.h>
 
 // VendorId of the Endpoint on the CastingPlayer that the CastingApp desires to interact with after connection
 const uint16_t kDesiredEndpointVendorId = 65521;
 
 DiscoveryDelegateImpl * DiscoveryDelegateImpl::_discoveryDelegateImpl = nullptr;
+bool awaitingCommissionerPasscodeInput                                = false;
```

Comment:
nit: given that these are globals, we may want to prefix them as `g` like `gAwaitingCommissionerPasscodeInput` to make code slightly easier to read

Reply:
That will be good, changing.